src/pudl/package_data/eia923/column_maps/generation_fuel.csv

```diff
@@ -93,6 +93,6 @@ total_fuel_consumption_quantity,total_fuel_consumption_quantity,total_fuel_consu
 total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtus,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu,total_fuel_consumption_mmbtu
 elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtus,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu,elec_fuel_consumption_mmbtu
-net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours
+net_generation_mwh_total,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours,net_generation_megawatthours
```

Comment:
We're not actually keeping this column in the end, right?  If we are, I think the units should be the last part of the name. based on how we've named other columns and [our naming conventions](https://catalystcoop-pudl.readthedocs.io/en/nightly/dev/naming_conventions.html#columns-and-field-names)

> Append units to field names where applicable (e.g. net_generation_mwh). This includes “per unit” signifiers (e.g. _pct for percent, _ppm for parts per million, or a generic _per_unit when the type of unit varies, as in columns containing a heterogeneous collection of fuels)

Reply:
we are not keeping this column. but your right our conventions would imply `total_net_generation_mwh`. i think that makes sense.

Reply:
just going to flag that we should also hypothetically convert all of the columns that say `net_generation_mwh_<MONTH>` by that same logic, but def OOS for this project (also they get combined into `net_generation_mwh` anyways.

For now I will change the column to `total_net_generation_mwh`